sonar-scanner-engine/src/main/java/org/sonar/scanner/report/ActiveRulesPublisher.java

```diff
@@ -43,6 +43,7 @@ public void publish(ScannerReportWriter writer) {
         builder.setRuleRepository(input.ruleKey().repository());
         builder.setRuleKey(input.ruleKey().rule());
         builder.setSeverity(Constants.Severity.valueOf(input.severity()));
+        builder.setCreatedAt(input.createdAt());
```

Comment:
or you could use a method reference to `DefaultActiveRule.class::cast`

(much cleaner, imho)